test/tests/markers/reporter_point_marker/point_marker_test.i

```diff
@@ -0,0 +1,62 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 10
+  ny = 10
+  nz = 0
+  zmax = 0
+  elem_type = QUAD4
+[]
+
+[Problem]
+  solve = false
+[]
+
+[Executioner]
+  type = Steady
+[]
+
+[Reporters]
+  [coords]
+    type=ConstantReporter
+    real_vector_names = 'x y z'
+    real_vector_values = '.31 .41 .51  .31 .41 .51 .31 .41 .51 .8;
+                          .31 .31 .31 .41  .41 .41 .51 .51 .51 .8;
+                           0   0   0  0   0   0  0  0   0  1;'
+    outputs=none
+  []
+  [bad_coords]
+    type=ConstantReporter
+    real_vector_names = 'x y z'
+    real_vector_values = '.31 .41 .51;
+                          .31 .31 .31 .41  .41 .41 .51 .51;
+                           0   0   0  0   0   0  0  0   0  1;'
+    outputs=none
+  []
+[]
+
+[Adaptivity]
+  [Markers]
+      active = 'box'
+    [box]
+      type = ReporterPointMarker
+      x_coord_name = coords/x
+      y_coord_name = coords/y
+      z_coord_name = coords/z
+      inside = refine
+      empty = do_nothing
```

Comment:
If empty = coarsen, would it cause a conflicting issue? 
For example, if a quad element is refined into four sub elements as shown below,  then only top left element contains a point.  Now the top left element wants to refine, but others element want to coarsen.  What will happen? 

```
 -----
|x |  |
 -----
|  |  |
-----
```

Reply:
That is a general issue that all markers face. I believe libmesh resolves
that by giving higher priority to refinement.

Reply:
I added a nice test for this.  if ```inside=refine``` and ```empty=coarsen``` and the I have the case like you mention, it will not coarsen the refined mesh.  In the same test I also added a moving point and as the point moves from one element to another, its previous element does get properly coarsened.